src/hearings/containers/request-hearing/hearing-venue/hearing-venue.component.html

```diff
@@ -48,18 +48,18 @@ <h1 class="govuk-heading-l">
                       <span class="govuk-visually-hidden">Error:</span> Enter a valid location
                     </span>
                     <exui-search-location class="search-location"
-                      [selectedLocations$]="selectedLocations$"
+                      [selectedLocations]="selectedLocations"
                       [control]="findLocationFormGroup.controls.locationSelectedFormControl"
-                      [locations$]="locationsFound$"
+                      [locationsDisplayedInDrop]="locationsDisplayedInDrop"
```

Comment:
tricky name locationsDisplayedInDrop, can it just be locationDisplayed or displayedLocation?